File: Assignment_33/Program5.c
 #include<stdio.h>
 
 void DisplaySchedule(char chDiv)
{
    if(chDiv >= 'a' && chDiv <='z')
    chDiv = chDiv - 32;
    
        switch(chDiv)
        {
        case 'A':
            printf("Your Exam At 7 AM :\n");
            break;
        
        case 'B':
            printf("Your Exam At 8.30 AM :\n");
            break;

        case 'C':
            printf("Your Exam At 9.20 AM :\n");
            break;

        case 'D':
            printf("Your Exam At 10.30 AM :\n");
            break;

        default:
            printf("Invalid Division :\n");
        } 
         
}
int main()
{
     char cValue = '\0';
      
     printf("Enter Your Division :\n");
     scanf("%[^'\n']c",&cValue);

     DisplaySchedule(cValue);
     return 0;
}

File: Assignment_33/Program5.java
import java.util.*;

class DisplaySchedule
{
    public void ShowTime(char chDiv)
    {
         chDiv = Character.toUpperCase(chDiv);

        if(chDiv == 'A')
        {
            System.out.println("Your Exam At 7 AM :\n");
        }
        else if(chDiv == 'B')
        {
            System.out.println("Your Exam At 8.30 AM :\n");
        }
        else if(chDiv == 'C')
        {
            System.out.println("Your Exam At 9.20 AM :\n");
        }
        else if(chDiv == 'D')
        {
            System.out.println("Your Exam At 10.30 AM :\n");
        }
        else
        {
            System.out.println("Invalid Division :\n");
        }
        
    }   
}  
public class Program5
{
    public static void main(String[] args)
    {
    
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter Your Division:");
        char ch = sobj.next().charAt(0);

        DisplaySchedule dobj = new DisplaySchedule();
        dobj.ShowTime(ch);

    }
}
